widgets/src/main/java/org/pentaho/gwt/widgets/client/listbox/CustomListBox.java

```diff
@@ -922,9 +922,6 @@ public void onFocus( Widget widget ) {
   }
 
   public void onLostFocus( Widget widget ) {
-    if ( visible == 1 ) {
-      popup.hide();
-    }
```

Comment:
I hadn't noticed this, initially. Why remove the `onLostFocus` behavior of hiding the dialog?

Isn't the new key down behavior independent?
What use cases was this supposedly handling that are no longer valid?